### PR TITLE
Pop Up Window: Web Page Rendering Data Dynamically

### PR DESCRIPTION
**Pull request by abiola-arobieke**
In this milestone:

Pop up was created both for the desktop and mobile screen.
Dummy data was added to the index.js file and rendered dynamically on the portfolio section and in the pop up window.
Each "see project" button was made to render the exact content  that is particular to it on the pop up window .

### CODE REVIEW

**Review by OyePriscilla — COMMENTED** [review]
Hi @abiola-arobieke 👍 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You've have submitted a project that has no information about the **'Install', 'Usage' and 'Deployment'** sections of your README file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.js [hunk 1]
```diff
@@ -17,3 +78,89 @@ for (let i = 0; i < menuItems.length; i += 1) {
...
+    <ul class="d-flex space-x-24 p-highlight mb-32 project-summary">
+    </ul>
+    <p>
+      ${projects[i].description}
+    </p>
+    <ul class="d-flex space-x-1 list-tags list-style-n no-pad flex-wrap stack-list">
+    </ul>
+    <button class="btn-action mt-40 view-details" type="button">See project</button>
+  </div>
+  </div>
```
Comment (levy002) [inline index.js:106]: - [ ] Kindly make sure to add the horizontal line that separates the project stack and the two buttons.
- [ ] KIndly make the two buttons aligned correctly and if you see carefully, they touch both ends of their container. Kindly fix that.

   - ### YOUR DESIGN
   
     image

   - ### FIGMA DESIGN
   
     image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @abiola-arobieke   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Good UI :heavy_check_mark: 
- Professional PR :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi Team,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Descriptive PR ✔️ 
- Linters passing ✔️ 
- Popup working ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**